Packs/CortexXDR/Integrations/CortexXDRIR/README.md

```diff
@@ -3112,6 +3113,7 @@ Retrieve a list of the current user emails associated with one or more user grou
 
 ***
 Retrieve information about one or more roles created in the environment.
+Required license: Cortex XDR Pro per Endpoint, Cortex XDR Pro, or Cortex XDR Pro per GB.
```

Comment:
Please enforce the new line, You can see in the preview one line break is not shown.
https://stackoverflow.com/questions/36583502/how-to-force-a-linebreak